models/permission.go

```diff
@@ -50,8 +56,8 @@ const (
 	TAG_DELETE
 )
 
-func (r Permission) String() string {
-	return [...]string{
+func (r Permission) String() (string, error) {
```

Comment:
I changed this signature because I had a horrible 30min of debugging, finding the error in a panic trace when I forgot to add the new permissions added in here.